osi_django_app/lists/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class ListsConfig(AppConfig):
+    name = 'lists'
```

Comment:
Let's call the class `OpenSourceInternships-text`.

Reply:
I can't just rename the class name because it might break internal django configaration.
But I could create a new app called OpenSourceInternships and move all the files.I don't think there is a better a way to do this.

(Just to be clear the name of the app should be **OpenSourceInternships** and not  **OpenSourceInternships-/text/** right??)